src/Components/Auth/PrivateRoutes.tsx

```diff
@@ -0,0 +1,20 @@
+import { Location } from 'history';
+import { Navigate, Route } from 'react-router';
+
+interface Props extends Location {
+	state: { key: string };
+}
+export const PrivateRoutes = ({
+	path,
+	element,
+}: {
+	path: string;
+	element: JSX.Element;
```

Comment:
You can put the types in a different variable and then use the variable here for better code readability.

Reply:
yeah sure, will take care of it!